assets/src/edit-story/components/inspector/inspectorLayout.js

```diff
@@ -26,7 +26,7 @@ const InspectorBackground = styled.div`
 	grid-area: inspector;
 	background-color: ${ ( { theme } ) => theme.colors.fg.v1 };
 	height: 100%;
-	padding: 0 1em;
+	padding: 0;
```

Comment:
This change breaks styling on other panels. 
<img width="306" alt="Screenshot 2020-01-24 at 12 11 27" src="https://user-images.githubusercontent.com/237508/73068549-6860ff00-3ea3-11ea-8cf4-0bda84fd424e.png">

Reply:
I assume it this tab can now just use `SimplePanel` as well?

Reply:
Yes, forgot about this instance of the panel. Will fix!

Reply:
Fixed in 85ce8bd2e.

Reply:
Did you fix the prepublished panel as well?

Reply:
> Did you fix the prepublished panel as well?

Of course I totally did not just forget about that 🙄. Done in 7f9e4ea52